src/main/java/org/opensrp/queue/CustomPlanEvaluatorMessage.java

```diff
@@ -0,0 +1,26 @@
+package org.opensrp.queue;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+import org.smartregister.domain.Jurisdiction;
+import org.smartregister.domain.PlanDefinition;
+import org.smartregister.pathevaluator.TriggerType;
+
+import java.io.Serializable;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class CustomPlanEvaluatorMessage implements Serializable {
```

Comment:
Remove custom from the name of the class

Reply:
Done